DESCRIPTION

```diff
@@ -0,0 +1,13 @@
+Package: dashDangerouslySetInnerHtml
+Title: A dash component for specifying raw HTML
+Version: 0.0.3
+Description: A dash component for specifying raw HTML
+Depends: R (>= 3.0.2)
+Imports: 
+Suggests: dash
```

Comment:
I think we'll need to add an example within the package if we want to submit this to CRAN, and if that example requires `dashHtmlComponents`, we'll have to add it to the `Suggests:` field here.